apps/email/index.html

```diff
@@ -32,6 +32,7 @@
   <link rel="stylesheet" type="text/css" href="shared/style/progress_activity.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/tabs.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/value_selector.css" >
+  <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" >
```

Comment:
Let's separate the gaia-icons effort as a separate patch. I expect we may need to evaluate startup performance, and likely need some work to get some of the icons into the font. I would prefer to be able to land the rtl changes though quickly, without needing to wait or be blocked by that work.

If that impacts what you can do for rtl though, please let me know.